pycomlink/tests/test_cmlh5_to_netcdf.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from pycomlink.io.cmlh5 import read_from_cmlh5
```

Comment:
not required anymore

Reply:
important, because this function uses Comlink objects which will not work in the future

Reply:
My mistake, it is not used in the testing, therefore will be delete in the next commit